README.md

```diff
@@ -1,3 +1,226 @@
+
+To begin, we'll require only the modules we need to get our component up and running:
+
+```javascript
+var React = require('tux/React');
+
+```
+
+One of the biggest philosophies behind Tux is modularity. Tux is all about requiring only the modules you need to make your application as performant as possible. Here, we require 'tux/React' to gain specific access to the Tux React object. If we needed to use a router or say, include some actions, we would simply require 'tux/Router' and 'tux/Actions', respectively.
+
+To learn more about Tux modularity, go [here]().
+
+Next, we'll create a Tux Mutable Class, which is a type of React class specifically designed to compare mutable traits to determine if a component should re-render. We'll then give it some mutable traits, which have been architected to trigger componentWillMount and shouldComponentUpdate life cycle events that re-render the component upon change.
+
+```javascript
+var ToDo = React.createMutableClass({
+  mutableTraits: {
+    props: 'text',
+    state: 'editing'
+  }
+});
+```
+
+To learn more about Tux Mutable Classes, go [here]().
+
+Lets say we wanted to be able to delete a to-do item on our list. Well, normally we would have to pass down this method from the parent-most component through every component in the chain until we reach the child. This repetition introduces the potential for bugs and requires a lot of similar code.
+
+To remedy this, Tux classes are built with knowledge about who their nearest owners and ownees are, and invoking this.nearestOwnerProps immediately gives us access to methods defined in the nearest owner component.
+
+```javascript
+  deleteMessage: function(e) {
+    e.preventDefault();
+    this.nearestOwnerProps.deleteMessage(this.props.todo.id);
+  }
+```
+
+To learn more about how props are passed between owner and ownees, go [here]().
+
+What if we wanted our to-do items to animate upon entering and exiting? Tux provides an entire library of plug-and-play animation that seamlessly integrate with any Tux or React components. Using these pre-built animations is as simple as requring the wrapped animations and plugging them into your components.
+
+Require the animations you want:
+
+```
+  var Fly = require('tux/Animations/Fly');
+  var FadeUp = require('tux/Animations/Fade/Up');
+```
+And plug them in:
+
+```javascript
+  render: function () {
+    return (
+      <Fly id='id'>
+        <Todo todo={todo} />
+      </Fly>
+    );
+  };
+```
+```javascript
+  render: function () {
+    return (
+      <FadeUp id='id'>
+        <Todo todo={todo} />
+      </FadeUp>
+    )
+  }
+```
+Want more flexibility in your animations? Tux animations enable you to easily modify any wrapped animation component, providing a semantic interface to toggle a number of default properties, as well as an option for you to pass in any custom CSS properties you desire.
```

Comment:
Don't need to go any further into animations at this point can cut the rest of this.